akka-docs/src/main/paradox/stream/operators/Source-or-Flow/watch.md

```diff
@@ -18,6 +18,18 @@ Watch a specific `ActorRef` and signal a failure downstream once the actor termi
 The signaled failure will be an @java[@javadoc:[WatchedActorTerminatedException](akka.stream.WatchedActorTerminatedException)]
 @scala[@scaladoc[WatchedActorTerminatedException](akka.stream.WatchedActorTerminatedException)].
 
+## Example
+
+An `ActorRef` can be can be watched and the stream will fail with `WatchedActorTerminatedException` when the
```

Comment:
This text doesn't describe what the example does. Something more realistic would be good, may a flow with `ask` which fails if the actor stops?

Use `@apidoc`?

It might make sense to write "Akka classic ActorRef". Is there a "new actors API" alternative?

Reply:
When thinking what the operator could be used for I couldn't come up with anything, and that's why I kept it minimal and unrealistic.

`ask` already has watch built in.

There is no `watch` operator for Typed, and I don't think we should add it if we don't see good use cases for it.

Maybe I came to think of one realistic usage now. Let's running a stream in an actor, but instead of using ActorContext materializer the stream is started via system materializer. Then instead of using KillSwitch and PostStop one can use `watch(self)` from the stream. I don't know? Maybe still a convoluted usage?

Reply:
It is hard to come up with small enough, yet relevant examples.